src/App.jsx

```diff
@@ -2,6 +2,7 @@ import './App.css';
 import { useState } from 'react';
 import LoginForm from './Login';
 import { getUser } from './apiCalls';
+import MenuBar from './components/layout/MenuBar.jsx';
```

Comment:
The app uses typescript and typescript jsx instead of javascript/jsx, please refactor your files to be fomatted as ts and tsx to prevent merge conflicts.